packages/backend/test/attesterDescription.test.mjs

```diff
@@ -0,0 +1,165 @@
+      method: 'post',
+      headers: headers,
+    }).then((r) => r.json())
+
+    expect(post.passed).to.be.false
+
+    const get = await fetch(url.toString(), {
+      method: 'get',
+      headers: { network: headers.network },
+    }).then((r) => r.json())
+
+    Object.entries(get).forEach(([_, v]) => {
+      expect(v).to.equal('')
+    })
+  })
+
+  it('should not update info with invalid url', async () => {
+    const hash = ethers.utils.solidityKeccak256(
+      ['uint256', 'string'],
+      [headers.nonce, headers.description]
+    )
+
+    headers.url = 'invalid url'
+    headers.signature = await wallet.signMessage(hash)
+
+    const url = new URL(`/api/about/${APP_ADDRESS}`, HTTP_SERVER)
+    const post = await fetch(url.toString(), {
+      method: 'post',
+      headers: headers,
+    }).then((r) => r.json())
+
+    expect(post.passed).to.be.false
+
+    const get = await fetch(url.toString(), {
+      method: 'get',
+      headers: { network: headers.network },
+    }).then((r) => r.json())
+
+    Object.entries(get).forEach(([_, v]) => {
+      expect(v).to.equal('')
+    })
+  })
+
+  it('should successfully update info with correct signature', async () => {
+    const hash = ethers.utils.solidityKeccak256(
+      ['uint256', 'string'],
+      [headers.nonce, headers.description]
+    )
+
+    headers.signature = await wallet.signMessage(hash)
+
+    const url = new URL(`/api/about/${APP_ADDRESS}`, HTTP_SERVER)
+    const post = await fetch(url.toString(), {
+      method: 'post',
+      headers: headers,
+    }).then((r) => r.json())
+
+    expect(post.passed).to.be.true
+
+    const get = await fetch(url.toString(), {
+      method: 'get',
+      headers: { network: headers.network },
+    }).then((r) => r.json())
+
+    Object.entries(get).forEach(([k, v]) => {
+      expect(headers[k]).to.equal(v)
+    })
+  })
+
+  it('should not update info with invalid network', async () => {
+    const hash = ethers.utils.solidityKeccak256(
+      ['uint256', 'string'],
+      [headers.nonce, headers.description]
+    )
+    headers.network = BlockExplorer.Mainnet
+    headers.signature = await wallet.signMessage(hash)
+
+    const url = new URL(`/api/about/${APP_ADDRESS}`, HTTP_SERVER)
+    const post = await fetch(url.toString(), {
+      method: 'post',
+      headers: headers,
+    }).then((r) => r.json())
+
+    expect(post.passed).to.be.false
+
+    const get = await fetch(url.toString(), {
+      method: 'get',
+      headers: { network: headers.network },
+    }).then((r) => r.json())
+
+    Object.entries(get).forEach(([_, v]) => {
+      expect(v).to.equal('')
+    })
+  })
+
+  it('contract should have a valid signature for each signature', async () => {
+    const accounts = await ethers.getSigners()
+    const hash = ethers.utils.solidityKeccak256(
+      ['uint256', 'string'],
+      [headers.nonce, headers.description]
+    )
+
+    await Promise.all(
+      accounts.map(async (contract) => {
```

Comment:
here should be `account` rather than `wallet`?

Reply:
the test has a bad description. it's just meant to make sure that the `recoverAddress` operates as expected, so it actually isn't needed